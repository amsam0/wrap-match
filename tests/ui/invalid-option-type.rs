fn main() {}

#[wrap_match::wrap_match(success_message = true)]
fn invalid_option_type_1() -> Result<(), CustomError> {
    Err(CustomError::Error)?;
    Ok(())
}

#[wrap_match::wrap_match(log_success = "true")]
fn invalid_option_type_2() -> Result<(), CustomError> {
    Err(CustomError::Error)?;
    Ok(())
}
