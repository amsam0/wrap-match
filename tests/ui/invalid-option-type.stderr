error: expected string literal
 --> tests/ui/invalid-option-type.rs:3:44
  |
3 | #[wrap_match::wrap_match(success_message = true)]
  |                                            ^^^^

error: expected boolean literal
 --> tests/ui/invalid-option-type.rs:9:40
  |
9 | #[wrap_match::wrap_match(log_success = "true")]
  |                                        ^^^^^^
